src/mui/detail/TabbedShowLayout.js

```diff
@@ -24,36 +24,38 @@ class TabbedShowLayout extends Component {
             basePath,
             translate,
             version,
+            styles,
         } = this.props;
         return (
-            <div style={divStyle} key={version}>
-                <Tabs
-                    value={this.state.value}
-                    onChange={this.handleChange}
-                    contentContainerStyle={contentContainerStyle}
-                >
-                    {React.Children.map(
-                        children,
-                        (tab, index) =>
-                            tab ? (
-                                <Tab
-                                    key={tab.props.value}
-                                    label={translate(tab.props.label, {
-                                        _: tab.props.label,
-                                    })}
-                                    value={index}
-                                    icon={tab.props.icon}
-                                >
-                                    {React.cloneElement(tab, {
-                                        resource,
-                                        record,
-                                        basePath,
-                                    })}
-                                </Tab>
-                            ) : null
-                    )}
-                </Tabs>
-            </div>
+            <Tabs
+                value={this.state.value}
+                onChange={this.handleChange}
+                tabItemContainerStyle={styles.tabs}
+                inkBarStyle={styles.inkBar}
```

Comment:
same for styles. this should be documented if someone wants to apply custom style to the Tabs